src/code/code_800EC960.c

```diff
@@ -4633,13 +4633,13 @@ void Audio_UpdateSceneSequenceResumePoint(void) {
     }
 }
 
-void func_800F5718(void) {
+void Audio_PlayWindmillBgm(void) {
     if (Audio_GetActiveSeqId(SEQ_PLAYER_BGM_MAIN) != NA_BGM_WINDMILL) {
         SEQCMD_PLAY_SEQUENCE(SEQ_PLAYER_BGM_MAIN, 0, 0, NA_BGM_WINDMILL);
     }
 }
 
-void func_800F574C(f32 scaleTempoAndFreq, u8 duration) {
+void Audio_SetSeqTempoAndFreq(f32 scaleTempoAndFreq, u8 duration) {
     if (scaleTempoAndFreq == 1.0f) {
         SEQCMD_RESET_TEMPO(SEQ_PLAYER_BGM_MAIN, duration);
```

Comment:
Oh true, this would actually reset the tempo of the bgm now and not when the fanfare is finished. I guess how that behaves depends on whether they mute or stop the main bgm